app/storage.js

```diff
@@ -19,19 +20,19 @@ const initialiseContainers = async () => {
   if (config.createContainers) {
     console.log('Making sure blob containers exist')
     await container.createIfNotExists()
+    console.log('Containers ready')
   }
-  await initialiseFolders()
+  foldersInitialised ?? await initialiseFolders()
```

Comment:
Original setup means every blob interaction re-uploads default files.  Hits performance and adds no value